Source/Scene/Cesium3DTile.js

```diff
@@ -61,30 +67,69 @@ define([
         var contentHeader = header.content;
 
 // TODO: For the 3D Tiles spec, we want to change this to basically (x, y, z) to (x, y, z)
-        var b = header.box;
-        var rectangle = new Rectangle(b[0], b[1], b[2], b[3]);
-
-        this._tileBoundingBox = new TileBoundingBox({
-            rectangle : rectangle,
-            minimumHeight : b[4],
-            maximumHeight : b[5]
-        });
-        this._orientedBoundingBox = OrientedBoundingBox.fromRectangle(rectangle, b[4], b[5]);
+        var tileBoundingVolume = header.boundingVolume;
+        if (defined(tileBoundingVolume.box)) {
```

Comment:
I'm writing the spec now and I am a bit confused about `box`.  I thought the intention was for it to be an arbitrary object-oriented bounding box.  It would be an array of 12 numbers, with the first three being the center (x, y, z) and the next three being the columns for the half axes (x, y, z) similar to:

https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/OrientedBoundingBox.js#L57